application/controllers/admin/export.php

```diff
@@ -1327,6 +1335,79 @@ public function quexml($iSurveyID)
         }
     }
 
+
+    /**
+     * Get a Zipped version of survey printable questionnaires in all languages
+     * (including the template html assets)
+     *
+     * @param integer $iSurveyID
+     */
+    private function _exportPrintableHtmls($iSurveyID){
+        $oSurvey = Survey::model()->findByPk($iSurveyID);
+        $oTemplate = Template::model()->getInstance($oSurvey->template);
+        $assetsDir = Template::getTemplateURL($oSurvey->template);
+        $fullAssetsDir = Template::getTemplatePath($oSurvey->template);
+        $aLanguages = $oSurvey->getAllLanguages();
+        if(!empty($aLanguages)){
```

Comment:
In that case, why check it? You can use `assert` for mental checks/internal errors, but it's not really used in the LS code-base, might look weird.